spec/factories/factory.rb

```diff
@@ -1,6 +1,7 @@
 FactoryBot.define do
   factory :user do
     username "felix_pip"
-    email "felix@pip.com"
+    email "felix@email.com"
```

Comment:
Since both `username` and `email` need to be unique, you should use a sequence here.

Reply:
Fixed up commit (263fba0) in `--fixup` in sha: d234d15